audit_log/managers.py

```diff
@@ -0,0 +1,67 @@
+from typing import Optional, TYPE_CHECKING, Union
+
+from django.db import models
+from django.http import HttpRequest
+
+from audit_log.enums import Operation, Status
+from audit_log.services import audit_log_service
+from audit_log.types import AuditCommitMessage
+from audit_log.utils import create_commit_message, get_remote_address
+
+if TYPE_CHECKING:
+    from django.contrib.auth.models import AbstractUser
+
+
+class AuditLogQuerySet(models.QuerySet):
+    def with_audit_log(
+        self,
+        user: "AbstractUser",
+        operation: Operation,
+        status: Union[Status, str] = Status.SUCCESS.value,
+        ip_address="",
```

Comment:
There's batteries included in Python for IP addresses i.e. https://docs.python.org/3/library/ipaddress.html which could be used for IP address validation, perhaps typing (I didn't check what kind of types the library provides out of the box). Just letting you know / consider, not saying that it must be used in this PR.

Reply:
won't use it here this time